implementations/micrometer-registry-dynatrace/src/test/java/io/micrometer/dynatrace/v2/DynatraceExporterV2Test.java

```diff
@@ -231,27 +231,26 @@ void toFunctionCounterLine() {
     @Test
     void toFunctionTimerLine() {
         class TestClass {
-            long time() {
+            long timeSomething() {
                 return 3L;
             }
 
-            double total() {
+            double doSomething() {
                 return 5.3;
             }
 
             final TimeUnit unit = TimeUnit.MILLISECONDS;
         }
 
         TestClass tester = new TestClass();
-        FunctionTimer.builder("my.function.timer", tester, TestClass::time, TestClass::total, tester.unit).register(meterRegistry);
+        FunctionTimer.builder("my.function.timer", tester, TestClass::timeSomething, TestClass::doSomething, tester.unit).register(meterRegistry);
         FunctionTimer functionTimer = meterRegistry.find("my.function.timer").functionTimer();
         assertNotNull(functionTimer);
 
-
         List<String> actual = exporter.toFunctionTimerLine(functionTimer).collect(Collectors.toList());
-        assertThat(actual).hasSize(2);
-        assertThat(actual.get(0)).startsWith("my.function.timer.count,dt.metrics.source=micrometer gauge,");
-        assertThat(actual.get(1)).startsWith("my.function.timer.sum,dt.metrics.source=micrometer gauge,");
+        assertThat(actual).hasSize(1);
+        assertThat(actual.get(0)).startsWith("my.function.timer,dt.metrics.source=micrometer gauge,");
+        assertThat(actual.get(0)).contains("min=").contains("max=").contains("sum=").contains("count=");
```

Comment:
Would it be possible to assert on the values as well? I suppose it should be min=max=sum=5.3 and count=3 here, right?